src/resmom_win/start_exec.c

```diff
@@ -1083,12 +1086,24 @@ set_homedir_to_local_default(job *pjob, char *username)
 			return ("");
 		}
 
-		strcpy(lpath, default_local_homedir(username,
-			pp->pw_userlogin, 0));
-		return (lpath);
+		if(default_local_homedir(username,
+			pp->pw_userlogin, 0) != NULL) {
+			strcpy(lpath, default_local_homedir(username,
+				pp->pw_userlogin, 0));
+			return (lpath);
+		} else {
+			log_event(PBSEVENT_ERROR, PBS_EVENTCLASS_JOB, LOG_ERR,
+				__func__, "Home directory was not found, job passed is NULL");
+		}
+		
+	}
+	
+	if(default_local_homedir(pjob->ji_wattr[(int)JOB_ATR_euser].at_val.at_str, pjob->ji_user->pw_userlogin, 0) != NULL)
+		strcpy(lpath, default_local_homedir(pjob->ji_wattr[(int)JOB_ATR_euser].at_val.at_str, pjob->ji_user->pw_userlogin, 0));
```

Comment:
Please dont call default_local_homedir() twice, instaed call once and save the ouput, verify and do strcpy